Clew/Controllers/ViewController.swift

```diff
@@ -292,6 +292,17 @@ class ViewController: UIViewController, ARSCNViewDelegate, SRCountdownTimerDeleg
         // render 3D keypoints
         renderKeypoint(keypoints[0].location)
         
+        // ? getting user location
+        prevKeypointPosition = getRealCoordinates(record: true)!.location
+        
+        // render path
+        if (showPath) {
+            renderPath(prevKeypointPosition, keypoints[0].location)
+        }
+        
+        // render pathpoints
+//        renderPathpoints(prevKeypointPosition, keypoints[0].location)
```

Comment:
It would be great to mark the commented out code so someone knows all of the changes to make to get the path points to render.  Even better would be to have a Boolean value in ViewController that controls whether to render the path points (for now the value could be hardcoded to false).